pyproject.toml

```diff
@@ -83,6 +83,7 @@ sacremoses = "0.1.1"
 hanzidentifier = "1.2.0"
 OpenCC = "1.1.9"
 PyICU = "2.8.1"
+fasttext = "*"
```

Comment:
Is this a mistake? I don't see where this is being used or is needed. Plus the version should be locked to a specific major/minor version so we don't have accidental breakage.

Reply:
It's needed in the tests. We can for sure lock it as well.

Reply:
Pinned, but there are a bunch of issues with fasttext: incompatibility with numpy v2, not compiling under poetry, it is unsupported now, and the repo is archived.  I used a prebuilt wheel here, the same as in hunspell/fastspell (we already install hunspell in taskcluster). We should have another look at all this in #248, I assume we'll switch to fastspell.